fury/shaders/visible_cross_section.glsl

```diff
@@ -0,0 +1,7 @@
+bool inVisibleCrossSection(vec3 center)
+{
+    bool xVal = center.x == crossSection.x;
+    bool yVal = center.y == crossSection.y;
+    bool zVal = center.z == crossSection.z;
+    return xVal || yVal || zVal;
+}
```

Comment:
This file should be in a folder determined by its function. Perhaps, `fury/shaders/interaction`, but I'll leave the name up to you.